tools/make-nodejs.sh

```diff
@@ -52,6 +52,7 @@ node -pe "JSON.stringify(fs.readFileSync('$THIRDPARTY/easylist.txt', 'utf8'))" \
 node -pe "JSON.stringify(fs.readFileSync('$THIRDPARTY/easyprivacy.txt', 'utf8'))" \
     > $DES/data/easyprivacy.json
 
+cp platform/nodejs/.*.json   $DES/
```

Comment:
The file has been renamed to `.eslintrc.json` so it gets picked up automatically by ESLint, since that's the default name for the configuration file.